cortx-monitorQuickstartGuide.md

```diff
@@ -137,23 +137,80 @@ This guide provides a step-by-step walkthrough for getting you CORTX-Monitor Ser
     -rw-r--r-- 1 root root   9592 Aug 18 07:27 cortx-libsspl_sec-method_pki-1.0.0-1_git8907300.el7.x86_64.rpm
     -rw-r--r-- 1 root root 101004 Aug 18 07:27 cortx-libsspl_sec-devel-1.0.0-1_git8907300.el7.x86_64.rpm
     ```
-
-3. Install CORTX-Monitor RPMs
+    
+5. Install CORTX-Monitor RPMs
 
    You'll need to create a repository and copy all the required RPMs to it. Run the commands:
 
     ```shell
-    
     $ mkdir MYRPMS
     $ cp -R /root/rpmbuild/RPMS/x86_64/cortx-libsspl_sec-* /root/rpmbuild/RPMS/noarch/cortx-sspl-* MYRPMS
-    $ find MYRPMS -name \*.rpm -print0 | sudo xargs -0 yum install -y
+    $ CORTX_MONITOR_BASE_URL="https://raw.githubusercontent.com/Seagate/cortx-monitor/main"
+    $ curl $CORTX_MONITOR_BASE_URL/low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy -o sspl_dev_deploy
+    $ chmod a+x sspl_dev_deploy
```

Comment:
keep a clear n bold note that this script is for dev purpose, to independently install SSPL only, along with its dependencies and conflicts with CORTX Provisioning steps meant for whole CORTX stack deployment.

Reply:
>conflicts with CORTX Provisioning steps meant for whole CORTX stack deployment.

this part is not stated